Resources/Prototypes/Entities/Clothing/Shoes/misc.yml

```
@@ -62,17 +63,18 @@
     sprite: Clothing/Shoes/Misc/lizard-slippers.rsi
   - type: FootstepModifier
     footstepSoundCollection:
-      collection: FootstepCarpet
+      collection: FootstepLizardSlippers
       params:
-        variation: 0.02
+        variation: 0.06
+        volume: -10
   - type: Construction
     graph: ClothingShoeSlippersLizard
     node: shoes
   - type: Edible
     useSound:
       path: /Audio/Items/Toys/weh.ogg # Copied from lizard plush sound
       params:
-        volume: -5
+        volume: -10
```

```suggestion
        volume: -5
```
imo leave this -- they can only be eaten like three times.